protocol.md

```diff
@@ -190,33 +190,39 @@ Cryptographic algoritms used by PFS are described in [Perfect forward secrecy se
 ## Topic
 
 There are two types of Whisper topics the protocol uses:
-* static topic for `user-message` message type (also called _contact discovery topic_)
-* dynamic topic based on a chat name for `public-group-user-message` message type.
+* topics for private chats (also called _partitioned topics_)
```

Comment:
I think phrasing this proposal as a change to the current one makes more sense than editing it in line. I.e. we propose improval to current spec (draft) that changes X to Y for Z reasons this will have W implications. Other solutions XX to problem were considered, etc.